.travis.yml

```diff
@@ -49,6 +49,16 @@ jobs:
         - docker push "quay.io/aelve/guide:$BRANCH--back"
       if: type = push
 
+    - stage: "Test haddocked <= 251"
```

Comment:
This is not easy to understand for someone who has no idea what this code is supposed to do. It's nicer to spell out the error message, e.g. "FAIL: the limit is 251 undocumented top-level definitions, but $notHaddockedAfter were found".

It would also be nice to add a comment before `- stage: Test haddocked <= 251`, saying that we're trying to get the number of undocumented definitions to zero.

"haddocked" sounds pretty weird, though technically it's correct. I would recommend "documented" (or "undocumented" in this case).